api/v1/ytsaurus_types.go

```diff
@@ -355,6 +355,9 @@ type HTTPProxiesSpec struct {
 	HttpsPort     *int32 `json:"httpsPort,omitempty"`
 	HttpNodePort  *int32 `json:"httpNodePort,omitempty"`
 	HttpsNodePort *int32 `json:"httpsNodePort,omitempty"`
+	//+kubebuilder:default:=8123
+	ChytHttpServerPort     *int32 `json:"chytHttpServerPort,omitempty"`
+	ChytHttpServerNodePort *int32 `json:"chytHttpServerNodePort,omitempty"`
```

Comment:
Can we add https too?

Maybe it would be better to group options for http server into structure and use it embedded and as "chyt" field:

```
 httpPort: ..
 httpsPort: ..
 chyt:
   httpPort: ..
   httpsPort ..
```

Reply:
Yeah. https is already here

```
    NHttp::TServerConfigPtr HttpServer;
    NHttps::TServerConfigPtr HttpsServer;
    NHttp::TServerConfigPtr TvmOnlyHttpServer;
    NHttps::TServerConfigPtr TvmOnlyHttpsServer;
    NHttp::TServerConfigPtr ChytHttpServer;
    NHttps::TServerConfigPtr ChytHttpsServer;
```

Reply:
I fixed it, but chyt -> chytServer, is it ok?